src/vnet/devices/memif/memif.c

```diff
@@ -248,7 +295,8 @@ memif_process (vlib_main_t * vm, vlib_node_runtime_t * rt, vlib_frame_t * f)
   memif_if_t *mif;
   struct sockaddr_un sun;
   int sockfd;
-  f64 timeout = 3153600000.0 /* 100 years */ ;
+#define UNREACHABLE_TIMEOUT  3153600000.0	/* 100 years */
+  f64 timeout = UNREACHABLE_TIMEOUT;
```

Comment:
This whole timeout needs to disappear. look at l2fib_mac_age_scanner_process in src/vnet/l2/l2_fib.c.

Add something like 

        if (enabled)
          vlib_process_wait_for_event_or_clock (vm, 60 - last_run_duration);
        else
          vlib_process_wait_for_event (vm);

and send START event to process after first interface is created. Send STOP event after last interface is deleted.